doc/deployment.md

```diff
@@ -173,26 +173,25 @@ language, plus a language-agnostic clone container image. These images are
 necessary to run any tests that do not use [pre-built images][], such as the
 [examples][].
 
-The complete set of images built for `v1.5.1` is as follows:
+The complete set of images built for `v1.5.7` is as follows:
 
 ```shell
-gcr.io/grpc-testing/e2etest/init/build/csharp:v1.5.1
-gcr.io/grpc-testing/e2etest/init/build/dotnet:v1.5.1
-gcr.io/grpc-testing/e2etest/init/build/node:v1.5.1
-gcr.io/grpc-testing/e2etest/init/build/php7:v1.5.1
-gcr.io/grpc-testing/e2etest/init/build/ruby:v1.5.1
-gcr.io/grpc-testing/e2etest/init/clone:v1.5.1
-gcr.io/grpc-testing/e2etest/init/ready:v1.5.1
-gcr.io/grpc-testing/e2etest/runtime/controller:v1.5.1
-gcr.io/grpc-testing/e2etest/runtime/cxx:v1.5.1
-gcr.io/grpc-testing/e2etest/runtime/dotnet:v1.5.1
-gcr.io/grpc-testing/e2etest/runtime/driver:v1.5.1
-gcr.io/grpc-testing/e2etest/runtime/go:v1.5.1
-gcr.io/grpc-testing/e2etest/runtime/java:v1.5.1
-gcr.io/grpc-testing/e2etest/runtime/node:v1.5.1
-gcr.io/grpc-testing/e2etest/runtime/php7:v1.5.1
-gcr.io/grpc-testing/e2etest/runtime/python:v1.5.1
-gcr.io/grpc-testing/e2etest/runtime/ruby:v1.5.1
+gcr.io/grpc-testing/e2etest/init/build/csharp:v1.5.7
```

Comment:
Here the location and name of the images has changed. 

1. `gcr.io/` becomes `us-docker.pkg.dev/`.
3. `init/build/` becomes `init-build-`.
4. `init-` becomes `init-`.
5.  `runtime/` is removed.